src/components/molecules/SpaceEditForm/SpaceEditForm.scss

```
@@ -24,6 +24,16 @@
     padding: $spacing--xl;
   }
 
+  &__input {
+    &-reactionMute {
```

No camel case, kebob here. 

We reserve the camels only for the B of B.E.M. Please also do not start any selector with `&-` (unless rare corner case demands) as people in a rush might identify it as the more common `&__` and it does add restriction to the class naming and discovery. 

There is not much to gain of keeping style DRY-est possible. Having only few elements, not really much of a help putting the `input` in there.

```suggestion
    &__input-reaction-mute {
```

---

removed ef337b8